src/indexPage/react-components/BotLanding.jsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import Carousel from './Carousel.jsx';
+import { translate } from '../../common/i18n';
+import { getOAuthURLDeriv } from '../../common/appId';
+// eslint-disable-next-line arrow-body-style
+const BotLanding = () => {
+    return (
+        <div>
+            <Hero />
+            <SectionOne />
+            <Carousel />
+            <SectionTwo />
+            <SectionThree />
+            <Footer />
+        </div>
+    );
+};
+
+export default BotLanding;
+
+
+
+
+const Hero = () => (
```

Comment:
can we move the component to separate files?
It's better to keep only 1 component in a file.